tests/Doctrine/Tests/ORM/Functional/Ticket/GH5763Test.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\ORM\Id\AssignedGenerator;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+class GH5763Test extends OrmFunctionalTestCase
+{
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->_schemaTool->createSchema([
+            $this->_em->getClassMetadata(GH5763Entity::class),
+        ]);
+    }
+
+    public function testIssue()
```

Comment:
Let's try to be more descriptive?
```suggestion
    public function testCacheIsBustedOnColumnChange()
```

Or is this the recommended format for tests starting with GH?